tools/src/format.rs

```diff
@@ -413,6 +419,37 @@ pub fn format_file_data(
     for (i, line) in output_lines.iter().enumerate() {
         let lineno = i + 1;
 
+        // Compute the coverage data for this line (if any)
+        let coverage_data: String = if let Some(ref coverage) = coverage {
+            // There's 2 levels of not having data for a line here:
+            // 1. We had no coverage data, coverage is None.  In that case,
+            //    we'll map to -5.
```

Comment:
Doesn't look like we actually use -5 anywhere, it just emits `cov-no-data` directly. So update the comment I guess?